src/common/i18n/locales/en-US.json

```diff
@@ -175,6 +176,15 @@
     "promote": "Promote",
     "boost": "Boost"
   },
+  "cross-post": {
+    "title": "Cross Post",
+    "community-label": "Where to share this post:",
+    "community-placeholder": "Select a Community",
+    "message-label": "Your message to the community:",
```

Comment:
Unnecessary line, also can be removed from UI. But bottom of message input, please have informational text:

`Rewards are automatically declined for the cross post and discussions under cross post will be different and will follow different community guidelines.`